<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SurfaceViewer" name="viewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Algorithm</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QuadOptions" name="quad_options" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Rendering</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="check_show_oracle">
          <property name="text">
           <string>Show Oracle</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_show_oracle_sharp_edge_sample">
          <property name="text">
           <string>Show Oracle Sharp Edge Sample</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_show_quad_surface">
          <property name="text">
           <string>Show Quad Surface</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_auto_recenter_when_load">
          <property name="text">
           <string>Auto recenter when load</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Triangle_Mesh"/>
    <addaction name="actionOpen_Quad_Mesh"/>
    <addaction name="actionSaveQuad"/>
    <addaction name="actionExport_Quad_to_OBJ"/>
    <addaction name="actionSave_Quad_Sharp_Edge_Candidate"/>
    <addaction name="actionLoad_Quad_Sharp_Edge_Candidate"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionConvert_OBJ_to_Quad"/>
    <addaction name="actionConvert_OBJ_to_Off"/>
    <addaction name="actionBatch_Runner"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRotation_Center"/>
   <addaction name="actionRecord"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Oracle Mesh</string>
   </property>
  </action>
  <action name="actionSaveQuad">
   <property name="text">
    <string>Save Quad</string>
   </property>
  </action>
  <action name="actionOpen_Triangle_Mesh">
   <property name="text">
    <string>Open Triangle Mesh</string>
   </property>
  </action>
  <action name="actionOpen_Quad_Mesh">
   <property name="text">
    <string>Open Quad Mesh</string>
   </property>
  </action>
  <action name="actionConvert_OBJ_to_Quad">
   <property name="text">
    <string>Convert OBJ to Quad</string>
   </property>
  </action>
  <action name="actionConvert_OBJ_to_Off">
   <property name="text">
    <string>Convert OBJ to OFF</string>
   </property>
  </action>
  <action name="actionRotation_Center">
   <property name="text">
    <string>Rotation Center</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionSave_Quad_Sharp_Edge_Candidate">
   <property name="text">
    <string>Save Quad Sharp Edge Candidate</string>
   </property>
  </action>
  <action name="actionLoad_Quad_Sharp_Edge_Candidate">
   <property name="text">
    <string>Load Quad Sharp Edge Candidate</string>
   </property>
  </action>
  <action name="actionBatch_Runner">
   <property name="text">
    <string>Batch Runner</string>
   </property>
  </action>
  <action name="actionExport_Quad_to_OBJ">
   <property name="text">
    <string>Export Quad to OBJ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SurfaceViewer</class>
   <extends>QWidget</extends>
   <header>viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuadOptions</class>
   <extends>QWidget</extends>
   <header>../quad/quad_options.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
